cmd/start.go

```diff
@@ -46,6 +47,32 @@ This command will start a stack and run it in the background.
 			}
 		}
 
+		dockerCmd := exec.Command("docker", "-v")
+		stdout, err := dockerCmd.Output()
+
+		if err != nil {
+			fmt.Println(err.Error())
+			return err
```

Comment:
We probably don't want to print and return the error from the command here. The user likely won't know what to do with that. Instead, we should print something like "An error occurred running docker. Is docker installed on your computer?"

Reply:
I moved all the docker checks to a common file whose sole function is called at the beginning of all the mentioned files. It'll return the respective error based on the status of the docker/docker-compose checks

Reply:
`
firefly-cli rakshitakaulgud$ ff start newone
Running docker pre checks
Error: An error occurred while running docker. Is docker running on your computer?
Usage:
  ff start <stack_name> [flags]

Flags:
  -h, --help          help for start
  -n, --no-pull       Do not pull latest images when starting
  -b, --no-rollback   Do not automatically rollback changes if first time setup fails

Global Flags:
      --ansi string   control when to print ANSI control characters ("never"|"always"|"auto") (default "auto") (default "auto")
  -v, --verbose       verbose log output

Error: An error occurred while running docker. Is docker running on your computer?`

``
firefly-cli rakshitakaulgud$ ff remove newone
Running docker pre checks
WARNING: This will completely remove your stack and all of its data. Are you sure this is what you want to do?
completely delete FireFly stack 'newone' [y/N] y
reading stack config... done
deleting FireFly stack 'newone'... done